src/wazuh_modules/wm_aws.c

```diff
@@ -72,23 +72,60 @@ void* wm_aws_main(wm_aws *aws_config) {
         time_start = time(NULL);
 
         for (cur_bucket = aws_config->buckets; cur_bucket; cur_bucket = cur_bucket->next) {
-            if (cur_bucket->aws_account_id && cur_bucket->aws_account_alias) {
-                mtinfo(WM_AWS_LOGTAG, "Executing Bucket Analysis: %s (%s)", cur_bucket->aws_account_alias, cur_bucket->aws_account_id);
-            } else if (cur_bucket->aws_account_id) {
-                mtinfo(WM_AWS_LOGTAG, "Executing Bucket Analysis: %s", cur_bucket->aws_account_id);
-            } else {
-                mtinfo(WM_AWS_LOGTAG, "Executing Bucket Analysis: %s", cur_bucket->bucket);
+            if (cur_bucket->bucket && cur_bucket->trail_prefix && cur_bucket->aws_account_id && cur_bucket->type &&
```

Comment:
This block has a lot of repeated code. We should group the if's by fields to reduce the amount of lines.